e2e_samples/dataset_versioning/insert_sql/process.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+from datetime import datetime
+
+
+class Process:
+    """Process object to filter df with specified version
+    """
+
+    def __init__(self, df: pd.DataFrame, version_date='issue_id'):
+        self.VERSION_DATE = version_date
```

Comment:
does this VERSION_DATE align with python naming convention?

Reply:
it was constant before but not anymore. Good catch.